bundles/org.openhab.binding.tradfri/src/main/resources/ESH-INF/i18n/tradfri_de.properties

```diff
@@ -1,41 +1,45 @@
 # binding
-binding.tradfri.name = TR�DFRI Binding
-binding.tradfri.description = Dieses Binding integriert das IKEA TR�DFRI System. Durch dieses k�nnen die TR�DFRI Lampen und Leuchten gesteuert werden. 
+binding.tradfri.name = TRÅDFRI Binding
+binding.tradfri.description = Dieses Binding integriert das IKEA TRÅDFRI System. Durch dieses können die TRÅDFRI Lampen und Leuchten gesteuert werden. 
```

Comment:
The changes in this file (apart from the obviously blind-related ones) result from me storing the file with UTF8-encoding, as the special characters where not shown on my PC.

Reply:
That's wrong. Since we use Java 8 the `.properties` files need ISO-8859-1 encoding, please revert that.

Reply:
Thanks for the information, committed with ISO 8859-1.